packages/nextjs/components/impact-vector/ImpactVectorGraph.tsx

```diff
@@ -144,7 +139,15 @@ export default function ImpactVectorGraph({ data }: { data: DataSet[] }) {
               }
               return null;
             })}
-        </LineChart>
+          <defs>
+            <linearGradient id="colorTotal" x1="0" y1="1" x2="0" y2="0">
+              <stop offset="5%" stopColor="rgba(20, 124, 73, 0.1)" stopOpacity={0.8} />
+              <stop offset="95%" stopColor="#FBDD5D" stopOpacity={0} />
+            </linearGradient>
```

Comment:
Here's the linear gradient definition, and here's the css for the gradient from figma:
```css
background: linear-gradient(180deg, rgba(20, 124, 73, 0.1) 0%, #FBDD5D 100%);
```